app/views/journalist/articles/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with scope: :article, url: journalist_articles_path, local: true do |form| %>
```

Comment:
why is this added in this PR?

Reply:
it from an merged pr. i tried to solve the form_with issue and deleted and added links. maybe thats why it shows as added.

Reply:
out of scope for this pr, add this to the chore of moving create to the new journalist controller.

Reply:
done